website/content/docs/concepts/domain-model/credential-libraries.mdx

```diff
@@ -7,39 +7,70 @@ description: |-
 
 # Credential Libraries
 
-A credential library is a resource
-that provides [credentials][]
-of the same type and same access level
-from a single [credential store][].
+A credential library is a resource that provides [credentials][] of the same type and same access level from a single [credential store][].
```

Comment:
Can we keep the [semantic line breaks](https://sembr.org/)? It makes reading future diffs a lot easier.

Reply:
Sorry about that! Will do in the future.